### PR TITLE
Refactored elastic and neo4j configs into default.application.properties

### PR DESCRIPTION
**Pull request by simbamarufu1**
I also removed some dead code from ElasticConfig

### CODE REVIEW

File: gtas-parent/gtas-commons/src/main/java/gov/gtas/repository/AppConfigurationRepository.java [hunk 1]
```diff
@@ -44,7 +42,6 @@
 	String MAX_FLIGHTS_SAVED_PER_BATCH = "MAX_FLIGHTS_PER_BATCH";
 	String DATA_MANAGEMENT_TRUNC_TYPE_FLAG = "DATA_MANAGEMENT_TRUNC_TYPE_FLAG";
 	String DATA_MANAGEMENT_CUT_OFF_TIME_SPAN = "DATA_MANAGEMENT_CUT_OFF_TIME_SPAN";
-	String GRAPH_DB_URL = "GRAPH_DB_URL";
 	String GRAPH_DB_TOGGLE = "GRAPH_DB_TOGGLE";
```
Comment (originalname51) [inline gtas-parent/gtas-commons/src/main/java/gov/gtas/repository/AppConfigurationRepository.java]: The toggle needs to be moved over as well.

File: gtas-parent/gtas-commons/src/main/resources/sql/gtas_data.sql [hunk 2]
```diff
@@ -67,7 +65,6 @@ insert into app_configuration (opt, val, description) values('DATA_MANAGEMENT_TR
 insert into app_configuration (opt, val, description)values ('THREADS_ON_RULES', '5', 'Number of threads on loader.');
 insert into app_configuration (opt, val, description)values ('FUZZY_MATCHING', 'true', 'Fuzzy matching toggle');
 insert into app_configuration (opt, val, description)values ('DATA_MANAGEMENT_CUT_OFF_TIME_SPAN', '6', 'Time in months past which we can truncate data from our database');
-insert into app_configuration (opt, val, description)values ('GRAPH_DB_URL', 'bolt://localhost:7687', 'Neo4J Address');
 insert into app_configuration (opt, val, description)values ('GRAPH_DB_TOGGLE', 'false', 'Neo4J Toggle');
```
Comment (originalname51) [inline gtas-parent/gtas-commons/src/main/resources/sql/gtas_data.sql]: GRAPH_DB_TOGGLE is the toggle for the graph rule engine and needs to be moved as well.

File: gtas-parent/gtas-commons/src/main/java/gov/gtas/config/ElasticConfig.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.core.env.Environment;
+
+import javax.annotation.Resource;
+
+@Configuration
+@ComponentScan("gov.gtas")
+@PropertySource("classpath:default.application.properties")
+@PropertySource(value = "file:${catalina.home}/conf/application.properties", ignoreResourceNotFound = true)
```
Comment (simbamarufu1) [inline gtas-parent/gtas-commons/src/main/java/gov/gtas/config/ElasticConfig.java]: @originalname51 @tsegay We will need this file to also have those properties in other environments going forward

File: gtas-parent/gtas-commons/src/main/java/gov/gtas/repository/AppConfigurationRepository.java (hunk 1 above)
Comment (originalname51) [reply gtas-parent/gtas-commons/src/main/java/gov/gtas/repository/AppConfigurationRepository.java]: The toggle is for the graph db rule engine, not for neo4j itself.

**Pull request merged**